src/core/prompts/system.ts

```diff
@@ -329,6 +343,15 @@ Tracking conventions discovered in the codebase.
 </tracking_conventions>
 </add_capture_calls>
 
+## list_feature_flags
```

Comment:
We can probably eventually loop through all the tool definitions and add them at once, filtering them out based on the chat mode (`ask`, `plan`, `act`)